Explorer/Assets/DCL/NameTags/NametagPlacementSystem.cs

```diff
@@ -27,6 +25,8 @@ public partial class NametagPlacementSystem : BaseUnityLoopSystem
         private readonly ChatEntryConfigurationSO chatEntryConfiguration;
         private readonly NametagsData nametagsData;
         private SingleInstanceEntity playerCamera;
+        private float distanceFromCamera;
+        private Dictionary<string, NametagView> activeNametags = new ();
 
         public NametagPlacementSystem(World world, IObjectPool<NametagView> nametagViewPool, ChatEntryConfigurationSO chatEntryConfiguration, NametagsData nametagsData) : base(world)
```

Comment:
Did you check the performance of this class? it does quite a lot of math and transforms updates per frame, if it starts to be sluggish I would add a `TODO: ` here that invites someone to Jobify it.
if it's not that slow and we have a bubble limit, we can ignore it, but if there isn't, imagine a situation where 100 users are spamming nonsense

Reply:
We would need to have 100 avatars close to us because at certain distance nametags are not rendered, and once we have easier ways to test it we will reduce even more the range of chat bubbles, also there is a setting to disable them from the chat if they get too annoying, anyway calulations are not that heavy, only impact is the alpha change of the text mesh pro.
Added a comment